Sources/VLCRendererDiscoverer.m

```diff
@@ -153,16 +154,45 @@ - (void)dealloc
     return [list copy];
 }
 
+- (VLCRendererItem *)discoveredItemsContainItem:(libvlc_renderer_item_t *)item
+{
+    for (VLCRendererItem *tmpItem in _items) {
+        if (!strcmp(libvlc_renderer_item_name(tmpItem.rendererItem), libvlc_renderer_item_name(item))
+            && !strcmp(libvlc_renderer_item_type(tmpItem.rendererItem), libvlc_renderer_item_type(item))) {
```

Comment:
this was very hard to read and I had a hard time seeing what is actually being checked for. 

Could you maybe write this instead?
Bool hasSameName = !strcmp(libvlc_renderer_item_name(tmpItem.rendererItem), libvlc_renderer_item_name(item));
Bool hasSameType = !strcmp(libvlc_renderer_item_type(tmpItem.rendererItem), libvlc_renderer_item_type(item));
if (hasSameName && hasSameType)

Reply:
I agree that it isn't the most readable code here, but it is C and I don't know if creating two variables will be worth in this case?